specs/linking-profiles/link-profiles.md

```diff
@@ -91,14 +91,14 @@ Example some normative text: <span class="conformance">OSLC Services **MUST** su
+In bi-dirctional linking scenarios both OSLC participants are aware of links across their owned resources and enable link visibility and navigation at each side. Nevertheless storing a link at both sides is considered a ill-practice as it is essentially replication of data. This may result in inconsistencies as links are updated or deleted, since maintaining consistency requires synchrnization across the providers on any update. Therefore the recommended practice is to store links on one of the participants, and use link discovery by the other participant. Therefore there need to be an agreed convention on which side should store the link. OSLC links have an incoming and outgoing sides, determined by the role of the link. Usully one side will have an active predicate name, for example "implements" and the other side will have a passive predicate name, in this example it would be "implemented by". The active side is also considered the outgoing side, and the passive the incoming side. The convention is that the link is stored with the resource on the outgoing side, i.e. the resource with the active predicate. The incoming side would discover the links with one of the discovery methods discussed in the following sections. Note the creation of the link can be initiated by both providers. In case that the link is intiated by the incoming side provider, it needs to store it with the resource on the outgoing side provider. This is discussed in the put on resources section. In addition, for historical reasons, certain OSLC providers such as IBM ELM, may have some variations of behaviors between configuration enabled and non-configuration enabled modes, where in non-configuration modes there may still be a usage of backlink storage. In addition, providers do not support configurations such as CM (change management providers) would have preference for link storage over configuration enabled providers, to prevent baselined links to refer to mutable entities.
```

Comment:
Okay, I ran it through ChatGPT and here are the suggestions:

> Here are the typos and deviations from the Microsoft Technical Writing Style Guide in the given text:
> 
> -  "bi-dirctional" should be "bi-directional."
> -  "OSLC" should be spelled out as "Open Services for Lifecycle Collaboration" at least once in the text, and then it can be abbreviated as "OSLC" afterward.
> -  "aware" should be "aware of."
> -  "enable" should be "enables."
> -  "at each side" should be "on each side."
> -  "a ill-practice" should be "an ill-practice."
> -  "synchrnization" should be "synchronization."
> -  "Usully" should be "Usually."
> -  "predicate" should be "predicate."
> -  "for example" should be "for example," (comma missing).
> -  "i.e." should be "i.e.," (comma missing).
> -  "intiated" should be "initiated."
> -  "put on resources section" should be "Put on Resources section."
> -  "OSLC providers such as IBM ELM" should be "OSLC providers, such as IBM ELM" (comma missing).
> -  "may still be a usage" should be "may still use."
> -  "providers do not support configurations such as CM" should be "providers, such as CM (change management providers), do not support configurations" (comma missing).
> -  "baselined" should be "baselined."

Reply:
@berezovskyi 

I went over your requests and did the ones I was able to . Here your request and what is done and what needs agreement.

"bi-dirctional" should be "bi-directional." - OK
"OSLC" should be spelled out as "Open Services for Lifecycle Collaboration" at least once in the text, and then it can be abbreviated as "OSLC" afterward. - OK
"aware" should be "aware of." - OK
"enable" should be "enables." - unsure
"at each side" should be "on each side." - OK
"a ill-practice" should be "an ill-practice." - OK
"synchrnization" should be "synchronization." - OK
"Usully" should be "Usually." - OK
"predicate" should be "predicate." - OK
"for example" should be "for example," (comma missing). - OK
"i.e." should be "i.e.," (comma missing). - weird but K
"intiated" should be "initiated." - OK
"put on resources section" should be "Put on Resources section." - OK
"OSLC providers such as IBM ELM" should be "OSLC providers, such as IBM ELM" (comma missing). - OK
"may still be a usage" should be "may still use." - Disagree this corrupts the meaning
"providers do not support configurations such as CM" should be "providers, such as CM (change management providers), do not support configurations" (comma missing). - Discuss
"baselined" should be "baselined." - Discuss

Reply:
Thank you, @rsjazz! I agree with your assessment, thanks for the fixes!